test/Jamfile

```diff
@@ -47,11 +47,11 @@ UBLAS_TESTSET_SPARSE_COO ?=
 
 
 # Project settings
-project
+project boost-ublas-test
     : requirements
        <define>BOOST_UBLAS_NO_EXCEPTIONS
        <toolset>vacpp:<define>"BOOST_UBLAS_NO_ELEMENT_PROXIES"
-       <toolset>gcc:<cxxflags>"-Wall -pedantic -Wextra"
+       <toolset>gcc:<cxxflags>"-Wall -pedantic -Wextra -Wno-unknown-pragmas -std=c++17"
```

Comment:
This looks wrong: You are adding `-std=c++17` if the compiler is `gcc`. You shouldn't change the flag (but filter on it in `test/tensor/Jamfile`, as you actually do now. :-) )

Reply:
true. changed it.